validator/remote/src/integration-test/java/tech/pegasys/teku/validator/remote/typedef/OkHttpValidatorTypeDefClientTest.java

```diff
@@ -110,12 +108,8 @@ void blockProductionFallbacksToNonBlindedFlowIfBlindedEndpointIsNotAvailable()
   void publishesBlindedBlockSszEncoded() throws InterruptedException {
     mockWebServer.enqueue(new MockResponse().setResponseCode(200));
 
-    final SignedBlockContainer signedBlockContainer;
-    if (specMilestone.isGreaterThanOrEqualTo(SpecMilestone.DENEB)) {
-      signedBlockContainer = dataStructureUtil.randomSignedBlindedBlockContents();
-    } else {
-      signedBlockContainer = dataStructureUtil.randomSignedBlindedBeaconBlock();
-    }
+    final SignedBlockContainer signedBlockContainer =
```

Comment:
Could be `SignedBeaconBlock`, to need to generalise anymore

Reply:
yup, good point